src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -34,7 +34,8 @@
      <ProjectCapability Include="
                           VB;
                           Managed;
-                          ClassDesigner;" />
+                          ClassDesigner;
+                          PreventAutomaticMyApplication;" />
```

Comment:
This is what opts us _out_ of the new system, so we don't break everyones existing SDK-style VB projects by showing them property page controls that may not work 100%